src/index.js

```diff
@@ -38,7 +38,7 @@ documentReady( function() {
         createChart.bar( properties );
       }
 
-      if ( type === 'map' ) {
+      if ( type === 'tile_map' ) {
```

Comment:
This is a Highcharts setting and should stay `map` http://api.highcharts.com/highmaps/series%3Cmap%3E